tools/pull_request_hooks/autoLabelConfig.js

```diff
@@ -0,0 +1,124 @@
+
+// File Labels
+//
+// Add a label based on if a file is modified in the diff
+//
+// You can optionally set add_only to make the label one-way -
+// if the edit to the file is removed in a later commit,
+// the label will not be removed
+export const file_labels = {
+	'GitHub': {
+		filepaths: ['.github'],
+	},
+	'SQL': {
+		filepaths: ['SQL'],
+	},
+	'Map Edit': {
+		filepaths: ['_maps'],
+	},
+	'Tools': {
+		filepaths: ['tools'],
+	},
+	'Config Update': {
+		filepaths: ['config', 'code/controllers/configuration/entries'],
+		add_only: true,
+	},
+	'Sprites': {
+		filepaths: ['icons'],
+		add_only: true,
+	},
+	'Sound': {
+		filepaths: ['sound'],
+		add_only: true,
+	},
+	'UI': {
+		filepaths: ['tgui'],
+		add_only: true,
+	}
+}
+
+// Title Labels
+//
+// Add a label based on keywords in the title
+export const title_labels = {
+	'Logging' : {
+		keywords: ['log', 'logging'],
+	},
+	'Removal' : {
+		keywords: ['remove', 'delete'],
+	},
+	'Refactor' : {
+		keywords: ['refactor'],
+	},
+	'Unit Tests' : {
+		keywords: ['unit test'],
+	},
+	'April Fools' : {
+		keywords: ['[april fools]'],
+	},
+	'Do Not Merge' : {
+		keywords: ['[dnm]', '[do not merge]'],
+	},
+	'GBP: No Update' : {
+		keywords: ['[no gbp]'],
+	},
+	'Test Merge Only' : {
+		keywords: ['[tm only]', '[test merge only]'],
+	},
+}
+
+// Changelog Labels
+//
+// Adds labels based on keywords in the changelog
+// TODO use the existing changelog parser
```

Comment:
this might never happen

Reply:
actually on second thought doesn't TGS have something that we can substitute in instead? i forget the specifics but i'm pretty sure TGS6 has a better changelog thing anyhow